console/services/app_actions/app_manage.py

```diff
@@ -243,7 +243,9 @@ def deploy(self, tenant, service, user, group_version, committer_name=None, oaut
                     return 400, "该组件构建源代码仓库类型不正确", ""
                 try:
                     git_url = instance.get_clone_url(service.git_url)
-                except NoAccessKeyErr:
+                except NoAccessKeyErr as e:
```

Comment:
This is still not a valid modification. The analysis is recommended from the OAuth plugin code.